azure_arc_k8s_jumpstart/azure_stack_hci/powershell/aks_hci_connect.ps1

```diff
@@ -0,0 +1,13 @@
+# <--- Change the following environment variables according to your Azure service principal name --->
+
+
+$clusterName = #<The name of your k8s cluster as it will be shown in Azure Arc>
+$resourceGroup = #<Azure resource group name>
+$location = #<Azure resource group location>
+$subscriptionId = #<Azure subscription Id>
+$appId = #<Your Azure service principal name>
+$password = #<Your Azure service principal password>
+$tenant = #<Your Azure tenant ID>
+
+Install-AksHciArcOnboarding -clusterName $clusterName -resourcegroup $resourceGroup -location $location -subscriptionid $subscriptionId -clientid $appId -clientsecret $password -tenantid $tenant
```

Comment:
add tag 'Project=jumpstart_azure_arc_k8s'

Reply:
apparently the cmdlet does not have the option